ex1/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import "fmt"
+
+
+
+func main() {
+
+
+var OS int
+var Ayo string
+var learner bool
+
+
+
+year := 2016
+month := "June"
+today := true
+
+
+
+fmt.Println(OS)
+fmt.Println(Ayo)
+fmt.Println(learner)
+
+
+newvar:= float32(3.14)
```

Comment:
You don't need to cast a value thats decimal as float32 since it is considered a generic float64 unless you desire the 32bit version for needed reasons.